public/javascripts/Admin/build/Admin.js

```diff
@@ -1,6 +1,211 @@
-function Admin (_, $, c3, turf) {
```

Comment:
These are from the merge from master. Shouldnt appear in new branch I am working on.